Sky Atlas/Sky Atlas.html

```diff
@@ -24712,39 +24759,69 @@ <h1>Agent Scope Database</h1>
         </tr>
         <tr>
           <td>
             <dfn>Root Edit Expert Advisor Review</dfn>
           </td>
           <td>Spark</td>
           <td>Core</td>
-          <td>A future iteration of the Spark Artifact will specify guidelines for obtaining specialized review of proposals requiring advanced technical or financial analysis.</td>
+          <td><p>A future iteration of the Spark Artifact will specify guidelines for obtaining specialized review of proposals requiring advanced technical or financial analysis.</p></td>
         </tr>
         <tr>
           <td>
             <dfn>Root Edit Proposal Review By Operational Facilitator</dfn>
           </td>
           <td>Spark</td>
           <td>Core</td>
-          <td>Within seven (7) days of the proposal being submitted, the Operational Facilitator must review the Root Edit Proposal for alignment. If the proposal is aligned, the Operational Facilitator must respond to the Forum post to announce their finding. In this Forum post, the Operational Facilitator must also confirm that the proposal is feasible for Operational GovOps to operationalize. If the proposal is misaligned, the Operational Facilitator must respond to the Forum post to announce their finding and provide the reasoning for it.</td>
+          <td><p>Within seven (7) days of the proposal being submitted, the Operational Facilitator must review the Root Edit Proposal for alignment.</p><p>If the proposal is aligned, the Operational Facilitator must respond to the Forum post to announce their finding. In this Forum post, the Operational Facilitator must also confirm that the proposal is feasible for Operational GovOps to operationalize.</p><p>If the proposal is misaligned, the Operational Facilitator must respond to the Forum post to announce their finding and provide the reasoning for it.</p></td>
         </tr>
         <tr>
           <td>
             <dfn>Root Edit Token Holder Vote</dfn>
           </td>
           <td>Spark</td>
           <td>Core</td>
-          <td>Where their review of the proposal results in a finding of alignment with the Sky Core Atlas and Spark Artifact, the Operational Facilitator next triggers a Snapshot poll to allow token holders to vote on the proposal. The poll is open for three (3) days. A poll must have at least 10% of the circulating token supply participating and must have 50% of votes in favor to be approved.</td>
+          <td><p>Where their review of the proposal results in a finding of alignment with the Sky Core Atlas and Spark Artifact, the Operational Facilitator next triggers a Snapshot poll to allow token holders to vote on the proposal. The poll is open for three (3) days. A poll must have at least 10% of the circulating token supply participating and must have 50% of votes in favor to be approved.</p></td>
```

Comment:
nit: You don't need p tags for a single line entry

```suggestion
          <td>Where their review of the proposal results in a finding of alignment with the Sky Core Atlas and Spark Artifact, the Operational Facilitator next triggers a Snapshot poll to allow token holders to vote on the proposal. The poll is open for three (3) days. A poll must have at least 10% of the circulating token supply participating and must have 50% of votes in favor to be approved.</td>
```